### DISCUSSION TITLE
LFX Mentorship '24 Term 01 (March - May) call for proposals

### DISCUSSION THREAD

**Discussion by aliok** [cncf/mentoring#1117] [Announcements]
Hello everyone!

The LFX Mentorship 2024 Term 01 March - May is open now open for project ideas:
https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2024/01-Mar-May

Project submission and application timeline:
* project proposal period: January 8 - 24
* mentee applications open: January 29 - February 13 (2 weeks)
* application review/admission decisions/HR paperwork: February 14 - February 27

We're looking forward to seeing all the project ideas you're interested in working on!

## Timeline

Mentorship duration - three months (12 weeks - full-time schedule)

| activity | date |
| --- | --- |   
| project proposals | January 8 - 24, 5:00 PM PDT |
| mentee applications open | January 29 - February 13, 5:00 PM PDT |
| application review/admission decisions/HR paperwork | February 13 - February 27, 5:00 PM PDT |
| Mentorship program begins with the initial work assignments | March 4 (Week 1) | 
| Midterm mentee evaluations and first stipend payments | April 10 (Week 6) |
| Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals | May 22, 5:00 PM PST (Week 12) |
| Last day of term | May 31 |

[original announcement email](https://lists.cncf.io/g/tag-cs-mentoring-wg/message/11)

**Comment by jaymorelli96** [comment]
Hi @aliok, I am feeling a bit overwhelmed with all the information regarding the _mentee_ application process. I **really** want to start contributing and giving back to the community (my resolution of 2024 😄). I would love to have a guidance on how I can improve my chance of being selected for this program.

I am a big advocate for Go and Vim and I am very interested in the CNCF projects. If anyone is interested in learning more about me, please check my personal website (https://www.jaymorelli.me/) as I have a quite detailed and lengthy _about_ page.

**Reply by aliok** [reply]
hey @jaymorelli96 

Once the project proposals period ends, we will upload these project ideas to LFX Mentorship platform: https://mentorship.lfx.linuxfoundation.org/#projects_all

Starting Jan 29, you will be able to apply using that platform. You will fill a form and the mentors will make the selection of the mentee from the applicants.

> I would love to have a guidance on how I can improve my chance of being selected for this program.

Sure, thanks for asking :)

I would recommend these:
- Read our guide: https://contribute.cncf.io/contributors/getting-started/
- Try the project of the community that you're interested in, in your local computer. Follow tutorials, watch videos, etc. to understand the project better. File issues along the way, if you find any bugs/problems.
- Start contributing and engaging a bit early so that the mentors know you in advance when they are making the selections.
- Help others in the community channels with their questions.
- If you're already in a community and contributing as an open source beginner, ask the community to propose an idea here in this program

**Reply by mercenarysre** [reply]
Hello @aliok thanks so much for your response. Please is it the duty of applicants(mentees) to do project proposals or the project community does that..thanks

**Reply by aliok** [reply]
@TomiwaAribisala-git creating project proposals is the duty of the project community

**Reply by Tyagiquamar** [reply]
@aliok Hello, I am an open-source contributor, (contributed to Cloudforet through LFX mentorship) interested in contributing to cncf.
But I am unable to access this Slack channel https://cloud-native.slack.com/archives/CT6CWS1JN

Can you guide me where I can connect to be guided for cncf LFX mentorship.

**Reply by aliok** [reply]
> @aliok Hello, I am an open-source contributor, (contributed to Cloudforet through LFX mentorship) interested in contributing to cncf.
> Can you guide me where I can connect to be guided for cncf LFX mentorship.

@Tyagiquamar If you were a LFX mentee previously, you cannot apply again.

https://docs.linuxfoundation.org/lfx/mentorship/mentees says "Not be a prior or an active participant in another Linux Foundation mentorship program."

However, I encourage you to contribute to CNCF projects outside of LFX mentorship. See my comment above (https://github.com/cncf/mentoring/discussions/1117#discussioncomment-8129422)

**Comment by antoninbas** [comment]
Hi - I am a maintainer for Project Antrea. This is our first time submitting proposals for the LFX mentorship program: https://github.com/cncf/mentoring/pull/1129

I am currently registering as a Mentor at https://mentorship.lfx.linuxfoundation.org/. Sine this is our first year participating as a project, we do not yet have a "LFX Mentorship" program for Antrea. My question is: should I go ahead and try to create it now, or will the CNCF do it based on #1129?

**Reply by aliok** [reply]
@antoninbas CNCF will do it for you.
We will create the projects in LFX Mentorship platform (https://mentorship.lfx.linuxfoundation.org/) and you will be invited as a mentor.

If you're able, feel free to join the information session we’re doing for mentors:

North America: [Jan 29, 2024 10:30 AM (Pacific Time)](https://zoom-lfx.platform.linuxfoundation.org/meeting/99504894798?password=54a6ec2f-d649-432b-a918-acd17720f163)
EMEA + APAC: [Jan 30, 2024 8:00 AM (UTC)](https://zoom-lfx.platform.linuxfoundation.org/meeting/93514971498?password=907d67b6-3e04-42d3-afdc-52d288122c6f)

**Reply by antoninbas** [reply]
Thank you! Yes, I am planning to join. I wasn't sure whether there was any action required before that (such as registering as a mentor).

**Comment by aliok** [comment]
I am closing this discussion as we're in the contributor application period now. See https://github.com/cncf/mentoring/discussions/1150
